app/db_utils.go

```diff
@@ -0,0 +1,160 @@
+package app
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/lib/pq"
+	"log"
+	"strconv"
+	"strings"
+)
+
+func connectDB() (*sql.DB, error) {
+	var db *sql.DB
+	connectString := `
+		user=postgres 
+		password=meme 
+		host=35.192.115.150 
+		dbname=meme-db 
+		sslmode=disable 
+	`
+	db, openErr := sql.Open("postgres", connectString)
+	if openErr != nil {
+		log.Fatal(openErr)
+		return db, openErr
+	}
```

Comment:
Add defer db.Close() immediately after where resource is successfully acquired to ensure resource release after panic.